test/e2e/framework/service_util.go

```diff
@@ -677,6 +694,7 @@ func (j *ServiceTestJig) AddRCAntiAffinity(rc *v1.ReplicationController) {
 		})
 }
 
+// CreatePDBOrFail returns a PodDisruptionBudget for the given ReplicationController.
```

Comment:
what conditions cause this to fail?

Reply:
This fails if a PDB isn't ready. So, add that to the comment of method.